app/team.go

```diff
@@ -377,7 +377,7 @@ func (a *App) UpdateTeamMemberSchemeRoles(teamId string, userId string, isScheme
 }
 
 func (a *App) sendUpdatedMemberRoleEvent(userId string, member *model.TeamMember) {
-	message := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_MEMBERROLE_UPDATED, "", "", userId, nil)
+	message := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_MEMBERROLE_UPDATED, "", "", "", nil)
```

Comment:
In a technical sense, this solves the problem, but as discussed, I'm wondering about both performance and security concerns. 

On the performance side, do role changes warrant broadcasting an event to 30,000 users just so someone in a dialog sees the new information?

On the security side, /should/ everyone know about everyone else's member roles? (That is, is this privileged information?)

Let's get @jespino and @jupenur for the respective opinions.

Reply:
> On the performance side, do role changes warrant broadcasting an event to 30,000 users just so someone in a dialog sees the new information?

I did mention that when I was talking to @hahmadia about this earlier. I don't think it should necessarily be an issue for performance or security, but it does feel like a lot to handle an uncommon use case.

Reply:
Roles themselves aren't particularly sensitive, but broadcasting this event to all members across all teams and channels would allow guest users to learn about activity on the server, user IDs, and team IDs -- which they otherwise wouldn't have access to. I think this needs a different approach.